README.md

```diff
@@ -79,7 +79,28 @@ rocker {
   }
 }
 
-rockerVersion = '1.2.2'  // optional
+rockerVersion = '1.3.0'  // optional
+```
+
+This is a sample configuretion in Kotlin DSL:
+
+```
+import nu.studer.gradle.rocker.RockerConfig
+
+plugins {
+    id("nu.studer.rocker") version "1.0.1"
+    id("java")
+}
+
+configure<NamedDomainObjectContainer<RockerConfig>>() {
+    create("main") {
+        setTemplateDir(file("src/rocker"))
+        setOutputDir(file("src/generated/rocker"))
+        setOptimize(true) // optional
+    }
+}
+
+extra["rockerVersion"] = "1.3.0"
```

Comment:
This is correct but could also be changed to:
```suggestion
val rockerVersion by extra("1.3.0")
```

Not related to this PR but why not provide this configuration as a proper property on the rocker extension?